__docs__/wonder-blocks-button/button-shared.argtypes.ts

```diff
@@ -0,0 +1,108 @@
+import {ArgTypes} from "@storybook/react";
+import {IconMappings} from "../wonder-blocks-icon/phosphor-icon.argtypes";
+
+export default {
```

Comment:
I noticed the description for the `type` prop renders weird because the description includes a `<form>` tag (This seems to be pre-existing though!)

![image](https://github.com/user-attachments/assets/c65358e3-59f2-46ba-a7cf-a537262995ea)